cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/templates/etl/common/ETLTemplate.java

```diff
@@ -78,7 +78,7 @@ public void configureAdapter(String adapterName, T config, AdapterConfigurer con
       Constants.Source.PLUGINTYPE, sourceConfig.getName(), sourcePluginId,
       PluginProperties.builder().addAll(sourceConfig.getProperties()).build());
     if (source == null) {
```

Comment:
The alignment seems correct. The line above slightly throws it off, so fixed that.